crates/miden-lib/asm/account_components/multisig_rpo_falcon_512.masm

```diff
@@ -70,6 +70,62 @@ proc.assert_new_tx
     # => []
 end
 
+#! Update threshold config and add / remove owners
+#!
+#! Inputs:  [NEW_THRESHOLD_CONFIG]
+#! Outputs: []
+#!
+export.update_signers_and_threshold
+    dupw
+    # => [NEW_THRESHOLD_CONFIG, NEW_THRESHOLD_CONFIG]
+
+    push.THRESHOLD_CONFIG_SLOT
+    # => [slot, NEW_THRESHOLD_CONFIG, NEW_THRESHOLD_CONFIG]
+
+    exec.account::set_item
+    # => [OLD_THRESHOLD_CONFIG, NEW_THRESHOLD_CONFIG]
+
+    dropw
+    # => [NEW_THRESHOLD_CONFIG]
+
+    # TODO assert threshold <= num_approvers
+    # TODO only enter loop if pub keys exist in AdviceMap
+
+    drop drop swap drop
+    # => [num_approvers]
```

Comment:
```suggestion
    dropw
    # => [NEW_THRESHOLD_CONFIG]
    # => [?, ?, num_approvers, ?]

    drop drop swap drop
    # => [num_approvers]
```

Nit: I think this would be more readable if we did something like this, i.e. show the layout of the word on the stack whenever we operate on it directly.